docs/conf/portland/2019/news/welcome.rst

```diff
@@ -0,0 +1,58 @@
+:template: {{year}}/generic.html
+
+.. post:: Nov 28, 2018
+   :tags: portland-2019, website, cfp, tickets
+
+Announcing Write the Docs Portland 2019 CFP & Website
+=====================================================
+
+Welcome to the official launch of `Write the Docs Portland 2019 <http://www.writethedocs.org/conf/portland/2019/>`_. We hope you're getting excited – we certainly are! It’s shaping up to be another wonderful year full of great conversations and interesting talks.
+
+Today we are announcing our `Call for Proposals <http://www.writethedocs.org/conf/portland/2019/cfp/>`_
+and `new website <http://www.writethedocs.org/conf/portland/2019/>`_.
+Speaking at Write the Docs is a fantastic way to share your ideas with our inspiring community of documentarians.
+If there is something you’d really like to see a talk on this year, submit a proposal on it, or refer someone else who would be good!
+
+Call for Proposals
+------------------
+
+Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
+We're looking for ideas and use-cases from all disciplines and roles, so whether you're a technical writer, developer, UX designer, community manage, or support professional who cares about content and communication, we want to hear from you!
+
+You can read our full `Call for Proposals <http://www.writethedocs.org/conf/portland/2019/cfp/>`__ on the website.
+The Call for Proposals will be open until **Midnight PST January 17, 2019**.
+
+Website
+-------
+
+As a part of our continuous optimization efforts, we are excited to introduce an even friendlier, more navigable online home for our event. We have a new `conference website <http://www.writethedocs.org/conf/portland/2019/>`_ design this year.
+
+As with all things on the web, there are likely to be a few lingering bugs with the design. Feel free to reach out to us by replying to the email, or emailing us at support@writethedocs.org if you see any issues!
+
+Tickets
+-------
+
+Tickets will officially go on sale in **January 2019**.
+
+In the meantime, you can see the ticket prices on our `Tickets <http://www.writethedocs.org/conf/portland/2019/tickets/>`_ page.
+As with the previous years, we have tiered pricing to make it fair for everyone. Please buy the ticket that best fits your situation.
+
+**Please note that ticket prices will increase by a small percentage next year.** It's been more than 2 years since the last increase, and we tried to calculate the absolute minimum amount we'd need to raise to keep up with increasing vendor and operating costs.
```

Comment:
This needs to be updated or probably just dropped. We don't need to highlight ticket price increases I don't think.